services/QuillLMS/client/app/bundles/Connect/components/renderForQuestions/feedback.tsx

```diff
@@ -63,6 +63,7 @@ class FeedbackComponent extends React.Component<any, any> {
         returnVal = (<p>Fill in the blank with the correct option.</p>);
         break;
       case "default-with-cues":
+        debugger;
```

Comment:
http://localhost:5000/connect?showTranslations=true#/play/lesson/-KTAQiTDo_9gAnk3aBG5 
this activity uses this code